spec/lib/search_works_marc_spec.rb

```diff
@@ -57,14 +57,4 @@
       end
     end
   end
-  describe 'matched vernacular' do
```

Comment:
I think we lost the line-note because it was on a commit that got rebased.

This particular test is being removed because it's testing the `MarcImprint` class which no longer exists.

In addition to that, this particular test is really attempting to test the vernacular field matching.  There are more generalizable tests against vernacular field matching in the new `MarcField` base class (which we'll hopefully slowly begin migrating everything to).